docs/aerodromes/classc/Sydney.md

```diff
@@ -203,17 +203,16 @@ Unless operationally required, aircraft shall be assigned the following runways
 | Aircraft tracking | Runway  |
 | ----------------| --------- |
 | via KADOM   | 16R/34L      |
-| via WOL | 16R/34L |
+| via WOL | 16R/**34R** |
 | via RIC| 16R/34L |
-| via PEGSU| 16R/34L |
 | via OLSEM | 16L/34R |
-| via KAMBA | 16L/34R |
 | Other aircraft: |
 | To the NORTH and EAST | 16L/34R |
-| To the SOUTH and WEST | 16R/34L |
+| To the SOUTH WEST | 16R/**34R** |
```

Comment:
Would probably lean towards having this back the way it was. I know we're catering for high traffic loads, but if these aircraft have to do an overhead departure, it'll mess things up even more, and we don't want this the default in low traffic situations (especially aircraft on the radar departure).

Reply:
So changing aircraft via WOL (and therefore assigned the MARUB SID via 34R, but leaving other aircraft "to the south west" on 34L?

Reply:
Yeah I'd support that